autocomplete.js

```diff
@@ -276,6 +276,16 @@ function Autocomplete(_settings) {
           },
           false
         );
+        window.addEventListener(
+          "scroll",
+          function acresize() {
```

Comment:
Would it make sense to share a function between the resize and scroll event listeners instead of copying and pasting the code?

Reply:
I would agree with this. Can you put the function above the event listeners and reference it?

Reply:
Shouldn't it be possible to move 
```
if (lastInput === false) {
    return;
}
```
inside of `align`. Then we don't need another method for this. (`() => _align(lastInput)` does the trick then)

Reply:
That makes sense to me -- we'd just have to move the declaration of lastInput a little higher in the code. @crhallberg, do you have any concerns about this?

Reply:
> we'd just have to move the declaration of lastInput a little higher in the code
That's not necessary if we use 
```
if (input === false) {
    return;
}
```
instead. Committed the simplified version.